server/db.js

```diff
@@ -18,22 +18,24 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-var Db = require('mongodb').Db;
-var Server = require('mongodb').Server;
-var logger = require('./log.js');
-var common = require('./common.js');
-var bcrypt = require('bcryptjs');
+const Db = require('mongodb').Db;
+const Server = require('mongodb').Server;
+const logger = require('./log.js');
+const common = require('./common.js');
+const bcrypt = require('bcryptjs');
 
-var DB_HOST = process.env.DB_HOST || 'localhost';
-var DB_PORT = process.env.DB_PORT || 27017;
-var DB_NAME = process.env.DB_NAME || 'quizzard';
+const DB_HOST = process.env.DB_HOST || 'localhost';
+const DB_PORT = process.env.DB_PORT || 27017;
+const DB_NAME = process.env.DB_NAME || 'quizzard';
 
-var db = new Db(DB_NAME, new Server(DB_HOST, DB_PORT));
+const db = new Db(DB_NAME, new Server(DB_HOST, DB_PORT));
 
-var nextQuestionNumber = 0;
 var usersCollection;
 var questionsCollection;
 var analyticsCollection;
+var settingsCollection;
+
```

Comment:
Extra line

Reply:
This extra line is to separate the variables